### ISSUE TITLE
Total canvas memory use exceeds the maximum limit (384 MB).

### ISSUE THREAD

**Issue by gguynn** [amcharts/amcharts5#234]
Hello dev,

**preface:**
We fully migrated from amcharts4 to amcharts5 and failed to test on mobile. Oops.  We assumed that the move to canvas-based was superior to SVG.  Apparently, there are limitations?  Thank you for your time in reviewing our issue.

**stack**
- React

**notes**
- we render many charts on the same page in some circumstances
- we dispose/cleanup unmounted components

**browser/os**
- Safari/iOS latest

**screenshot**
<img width="1792" alt="Screen Shot 2022-01-23 at 12 13 06 PM" src="https://user-images.githubusercontent.com/1015228/150690736-d87deef7-2c7f-4e38-8ea6-a505854a5818.png">

**Comment by zeroin** [comment]
But are the charts rendered? If you turn off the dev tools? There are reports which say that with dev tools opened Safari does not collect garbage properly.

**Comment by gguynn** [comment]
> But are the charts rendered? If you turn off the dev tools? There are reports which say that with dev tools opened Safari does not collect garbage properly. 

They are not rendered.  I pulled up dev tools to figure out why they weren't rendering as I initially considered a JS error on our end.

To clarify, the first couple from the landing page render.  Navigation to other pages that have charts fail to render the charts completely.  Assuming at some point I've exhausted memory.

--

I did some investigation with Chrome dev tools also, and discovered no memory leaks.  I think your dispose method works well-- tho I may not understand how canvas is reported in memory.

I've seen it mentioned that setting a canvas Width X Height of 1 during dispose cleared the memory usage.

--

Sorry for the chapter.  Thanks for your attention.
